### PR TITLE
Attempt to save some vertical space on slides

### PR DESCRIPTION
**Pull request by darnuria**
Tried to save some vertical space on ownership slides like so:
Before:
Screenshot_2020-11-22 Ownership Borrowing

After:
Screenshot_2020-11-22 Ownership Borrowing(1)

How did I do? Use some full-qualified imports, remove newline, sprinkle some `?` on advanced slides (might break pedagogic flow `expect` may be better than `match + panic` if `?` is not adapted.

On `8.rs` I rewrote a little bit to get something who pass compiling!

### CODE REVIEW

**Comment by darnuria** [comment]
I had some reserve on [2.rs](https://github.com/ferrous-systems/teaching-material/blob/master/presentations/ownership-borrowing-in-brief/2.rs#L8-L11)

And on [1.rs](https://github.com/ferrous-systems/teaching-material/blob/master/presentations/ownership-borrowing-in-brief/1.rs#L8-L11) because rewriting with `?` might lose people... 
Maybe rewriting as:
```
    let mut file = match file_create {  // <2>
        Ok(f) => f, // <3>
        Err(e) => return Err(e), 
    };
```
Or:
```
    let mut file = match file_create {  // <2>
        Ok(f) => f, // <3>
        error @ Err(e) => return error, 
    };
```
is better?

From my teacher/trainer perspective: gives opportunity to say in next slide "hey look remember that awkward code with returning the error to break out? We have `?` for that!/

File: presentations/ownership-borrowing-in-brief/1.rs [hunk 1]
```diff
@@ -1,8 +1,7 @@
 use std::fs::File;
 use std::io::Write;
-use std::io;
```
Comment (matklad) [inline presentations/ownership-borrowing-in-brief/1.rs:3]: `use std::io::{self, Write}` ?

Comment (darnuria) [reply presentations/ownership-borrowing-in-brief/1.rs:3]: Even better will definitively make a change. ;)

**Comment by darnuria** [comment]
Hi @matklad, sorry I get submerged by work, did some change based on your suggestion. :)

**Comment by amanjeev** [comment]
I am going to close this PR since this is pretty old and still in draft and the content has moved forward.

**Comment by darnuria** [comment]
No problem with that, was little bit lagging on this notification.

**Pull request closed without merging**